--- combined.fs
+++ combined.fs
@@ -12,11 +12,14 @@
 370 TI-84+ LCD                 375 TI-84+ Keyboard
 380 TRS-80 4P drivers
 395 Dan SBC drivers
+
+500 AVR        600 8086        700 6809       800 6502
 ( ----- 001 )
 \ Z80 port's Macros and constants. See doc/code/z80.txt
 : Z80A ASML 320 327 LOADR 310 LOAD ( HAL flow ) ASMH ;
 : Z80C 302 308 LOADR ; : Z80H 310 314 LOADR ;
 : TRS804PM 380 LOAD ;
+: XARCHM 501 LOAD 601 LOAD 701 LOAD 801 LOAD ;
 \ see comment at TICKS' definition
 \ 7.373MHz target: 737t. outer: 37t inner: 16t
 \ tickfactor = (737 - 37) / 16
@@ -998,15 +1001,17 @@
   DUP 4 = IF VD_CURCL [ VID_WDTH 28 - LITN ] > IF
     [ VID_WDTH 24 - LITN ] ELSE VD_CURCL 4 + THEN
     [*TO] VD_CURCL DROP 0 THEN DUP UNTIL ;
+( ----- 199 )
+
 ( ----- 000 )
 AVR MASTER INDEX
 
-301 AVR macros                 302 AVR assembler
-320 SMS PS/2 controller
+501 AVR macros                 502 AVR assembler
+520 SMS PS/2 controller
 
 ( ----- 001 )
-: AVRA ASML 302 312 LOADR ;
-: ATMEGA328P 315 LOAD ;
+: AVRA ASML 502 512 LOADR ;
+: ATMEGA328P 515 LOAD ;
 ( ----- 002 )
 \ AVR assembler. See doc/asm/avr.txt.
 \ We divide by 2 because each PC represents a word.
@@ -1427,16 +1432,18 @@
 ' BRNE AGAIN?, \ no? loop
 R16 $1 ANDI,
 RET,
+( ----- 099 )
+
 ( ----- 000 )
 8086 MASTER INDEX
 
-301 8086 boot code             306 8086 HAL
-311 8086 assembler             320 8086 drivers
+601 8086 boot code             606 8086 HAL
+611 8086 assembler             620 8086 drivers
 ( ----- 001 )
 \ 8086 macros
-: 8086A ASML 311 317 LOADR 306 LOAD ( HAL flow ) ASMH ;
-: 8086C 302 305 LOADR ;
-: 8086H 306 310 LOADR ;
+: 8086A ASML 611 617 LOADR 606 LOAD ( HAL flow ) ASMH ;
+: 8086C 602 605 LOADR ;
+: 8086H 606 610 LOADR ;
 1 VALUE JROPLEN -1 VALUE JROFF
 ( ----- 002 )
 \ 8086 boot code. PS=SP, RS=BP, IP=DX
@@ -1682,18 +1689,20 @@
 CODE _spit ( c )
   POPp, AH $0e MOVri, ( print char ) $10 INT, ;CODE
 : CELL! ( c pos -- ) 0 CURSOR! _spit ;
+( ----- 099 )
+
 ( ----- 000 )
 6809 MASTER INDEX
 
-301 6809 macros                302 6809 boot code
-306 6809 HAL                   311 6809 assembler
-320 TRS-80 Color Computer 2
+701 6809 macros                702 6809 boot code
+706 6809 HAL                   711 6809 assembler
+720 TRS-80 Color Computer 2
 ( ----- 001 )
 ( 6809 declarations )
-: 6809A ASML 311 318 LOADR 306 LOAD ( HAL flow ) ASMH ;
-: 6809C 302 305 LOADR ;
-: 6809H 306 310 LOADR ;
-: COCO2 320 322 LOADR ;
+: 6809A ASML 711 718 LOADR 706 LOAD ( HAL flow ) ASMH ;
+: 6809C 702 705 LOADR ;
+: 6809H 706 710 LOADR ;
+: COCO2 720 722 LOADR ;
 1 VALUE JROPLEN -1 VALUE JROFF
 ( ----- 002 )
 ( 6809 Boot code. IP=Y, PS=S, RS=U  ) HERE TO ORG
@@ -1964,15 +1973,17 @@
     SWAP $400 + C! ELSE 2DROP THEN ;
 : CURSOR! ( new old -- )
   DROP $400 + DUP C@ $40 XOR SWAP C! ;
+( ----- 099 )
+
 ( ----- 000 )
 6502 MASTER INDEX
 
-301 6502 macros and consts     302 6502 assembler
-310 6502 boot code
+801 6502 macros and consts     802 6502 assembler
+810 6502 boot code
 ( ----- 001 )
 \ 6502 macros and constants. See doc/code/6502.txt
-: 6502A ASML 302 305 LOADR ;
-: 6502C 310 311 LOADR ;
+: 6502A ASML 802 805 LOADR ;
+: 6502C 810 811 LOADR ;
 1 VALUE JROPLEN -1 VALUE JROFF
 ( ----- 002 )
 \ 6502 assembler, Addressing modes.
